---
layout: page
title: Published Datasets and Code Repositories
subtitle: Hypothesis Driven Research in Plant Genomics and Phenomics

googlefonts: ["Monoton", "Lobster"]
bigimg:
---

Citable datasets (some associated with publications or preprints and some stand alone).

* Grzybowski M, Mural RV, Xu G, Turkus J, Yang Jinliang, Schnable JC. (2023) A common
resequencing-based genetic marker dataset for global maize diversity. The Plant Journal.
doi: <a href="https://doi.org/10.1111/tpj.16123">10.1111/tpj.16123</a>. <b>Dryad.</b>
<a href="https://doi.org/10.5061/dryad.bnzs7h4f1">10.5061/dryad.bnzs7h4f1</a>

*Whole-genome resequencing data from 1276 previously published maize samples plus 239 newly
resequenced samples are merged into a unified marker set. The release includes approximately
366 million segregating variants and a high-confidence subset of about 46 million variants.*

* HIPS hybrid trial dataset. <b>GitHub.</b>
<a href="https://github.com/jdavis-132/hips">https://github.com/jdavis-132/hips</a>

*Trait and metadata for 122 maize hybrids in 34 environments across Iowa and Nebraska. Please
cite: Davis JM, Coffey LM, Turkus J, Lopez-Corona L, Linders K, Ullagaddi C, Santra DK,
Schnable PS, Schnable JC (2025) Assessing the impact of yield plasticity on hybrid performance
in maize. Physiologia Plantarum doi: <a href="https://doi.org/10.1111/ppl.70278">10.1111/ppl.70278</a>
(2022 Hybrid data citation). Shrestha N, Powadi A, Davis J, Ayanlade TT, Liu H, Tross MC,
Mathivanan RK, Bares J, Lopez-Corona L, Turkus J, Coffey L, Tubery TZ, Ge Y, Sarkar S,
Schnable JC, Ganapathysubramanian B, Schnable PS (2025) Plot-level satellite imagery can
substitute for UAVs in assessing maize phenotypes across multistate field trials. Plants Planet
People doi: <a href="https://doi.org/10.1002/ppp3.10613">10.1002/ppp3.10613</a> (2023 Hybrid
data citation).*

* Mural RV, Sun G, Grzybowski M, Tross MC, Jin H, Smith C, Newton L, Andorf CM, Woodhouse MR,
Thompson AM, Sigmon B, Schnable JC (2022) Association mapping across a multitude of traits
collected in diverse environments identifies pleiotropic loci in maize. GigaScience doi:
<a href="https://doi.org/10.1093/gigascience/giac080">10.1093/gigascience/giac080</a>.
<b>Supplementary tables.</b> <a href="/Pubs/Mural_2022_Gigascience_Supplementary_Table_S2.xlsx">Table S2</a>
<a href="/Pubs/Mural_2022_Gigascience_Supplementary_Table_S3.xlsx">Table S3</a>

*Supplementary Tables S2 and S3 compile trait measurements and metadata for maize association
panels (MAP, SAM, WiDiv), drawn from 16 published studies plus 2020 Lincoln, Nebraska
phenotypes. The tables cover 162 trait data sets across 7 states for 1,014 unique genotypes with
marker data, with S3 documenting trait definitions, collection sites, and source papers.*

* Schnable (2019) Pan-Grass Syntenic Gene Set (sorghum referenced) with both maize v3 and maize v4 gene models **figShare** doi: [10.6084/m9.figshare.7926674.v1](https://doi.org/10.6084/m9.figshare.7926674.v1)

*Updated syntenic gene list, now with pearl millet, teff, and a way WAY better version of the oropetium genome. Please cite <a href="http://dx.doi.org/10.1105/tpc.17.00354">Zhang et al 2017</a>.*

* Schnable JC (2018) Pan-Grass Syntenic Gene Set (sorghum referenced) with both maize v3 and maize v4 gene models <b>figShare.</b> <a href="https://doi.org/10.6084/m9.figshare.6974417.v1">10.6084/m9.figshare.6974417.v1</a>


*Syntenic gene list for a bunch of species. Splits out maize1 and maize2 subgenomes. Uses maize v3, maizev4 sorghum v3, setaria v2.2 and rice v7 (among others). Constructed using LastZ to identify homologous genes, QuotaAlign to identify syntenic regions and python polishing scripts to identify high confidence orthologs within syntenic blocks. Maize v3 and maize v4 are treated as two separate species for the purposes of this analysis. Please cite <a href="http://dx.doi.org/10.1105/tpc.17.00354">Zhang et al 2017</a>.*

* Liang Z, Schnable JC. (2017) RGB and Hyperspectral images of approx. 1 meter tall corn plants from the genomes to fields inbred panel. <b>Cyverse.</b> <a href="https://doi.org/10.7946/P22K7V">10.7946/P22K7V</a>

*A training dataset for developing methods for extracting new maize (corn) phenotypic measurements from different types of image data, testing the heritability of these measurements (the proportion of total variation explained by genetic differences), and searching for correlations between these measurements and agronomically relevant traits collected from the same genotypes in field trials across many states.*

* Schnable JC, Zhang Y, Ngu DWC. (2016) Pan-Grass Syntenic Gene Set (sorghum referenced). <b>figShare.</b> <a href="https://dx.doi.org/10.6084/m9.figshare.3113488.v1">10.6084/m9.figshare.3113488.v1</a><br>

*Syntenic gene list for a bunch of species. Splits out maize1 and maize2 subgenomes. Uses maize v3, sorghum v3, setaria v2.2 and rice v7 (among others). Constructed using LastZ to identify homologous genes, QuotaAlign to identify syntenic regions and python polishing scripts to identify high confidence orthologs within syntenic blocks.*

* Schnable JC, Lyons E. (2015) Plant Paleopolyploidy. <b>figShare.</b>
<a href="https://dx.doi.org/10.6084/m9.figshare.1538627.v1">10.6084/m9.figshare.1538627.v1</a><br>

*Best effort attempt to describe the positioning and evidence supporting different known whole genome events relative to the phylogeny of all plant genomes sequenced at the time. <a href="https://genomevolution.org/wiki/index.php/Plant_paleopolyploidy">Also on CoGePedia.</a>*

* Schnable, JC (2015) Maize and Sorghum Genesets. <b>figShare.</b>
<a href="https://dx.doi.org/10.6084/m9.figshare.1537501.v1">10.6084/m9.figshare.1537501.v1</a><br>

*BED files answering the question: Where would each sorghum gene be in each maize subgenome if there were zero fractionation after the maize WGD? Also some useful figures generated using the same data.*

* Schnable JC. (2015) Sequenced Plant Genomes. <b>CoGePedia</b> <a href="https://genomevolution.org/wiki/index.php/Sequenced_plant_genomes">Wiki Page</a><br>

*Exhaustive list of published plant genomes as of early 2015.*

# Code

<a href="https://github.com/alejandropages/SLHTP"> <b>SLHTP</b></a> <i>- A set of software packages for automated image processing based phenotyping </i>
<br> Created by Alejandro D Pages, undergraduate researcher, 2018-19.
<br> The Schnable Lab High Throughput Phenotyping (SLHTP) software package is written in python and houses the various scripts used in the development of three image processing software packages for automated phenotyping tasks.
The tools available are:
* <b>beanpheno</b> A tool that can be used to process images of beans to obtain phenotype information for use in GWAS
* <b>kernelpheno (incomplete)</b> All the scripts and notebooks that were used in the development of a method to rate popcorn kernels for vitreousness from images taken on a light box. The image data was not collected with the intention of applying automated image processing methods in mind and therefore were too inconsistent for a simple method to be applied with good results. However progress was made towards that goal and perhaps a new data collection effort is warranted for the completion of this project.
* <b>earpheno</b> A set of tools that are useful for backend data management of a project involving <a href="https://www.biorxiv.org/content/10.1101/384974v1"> PhytoMorph</a>, a tool for extracting phenotype relevant measurements of maize ears.
