---
layout: page
title: Schnable Lab Research Interests
---

Research in the Schnable Lab@UNL is an interdisciplinary endeavor. Here are three main clusters:
<ul>
<li><a href="#Comparative">Comparative functional genomics</a></li>
<li><a href="#Phenotyping">High throughput phenotyping</a></li>
<li><a href="#QuantGen">Quantitative genetics</a></li>
</ul>

<a id="Comparative">

## Comparative Functional Genomics


Research in this area focuses on utilizing cross species comparisons to separate out the functional and functionless portions of plant genomes and then predicting the functions of the apparently functional bits.

![CompGen](/images/optimized/Science_Images/CompGenMerged.jpg)

### Recent Lab Publications on Comparative Genomics

* **Raju SKK**, **Zhang Y**, Mahboub S, **Ngu DW**, Qiu Y, Harmon FG, **Schnable JC**, Roston RL (2024) "[Rhythmic lipid and gene expression responses to chilling in panicoid grasses.](https://doi.org/10.1093/jxb/erae247)" *Journal of Experimental Botany* doi: 10.1093/jxb/erae247
* **Sun G**, Yu H, Wase N, Su S, Jenkins J, Zhou B, **Torres-Rodriguez JV**, Zhuang W, **Ngu DW**, Birdseye D, Foltz A, **Sigmon B**, Yu B, Obata T, Schmutz J, **Schnable JC** (2022) "[Genome of Paspalum vaginatum and the role of trehalose mediated autophagy in increasing maize biomass.](https://doi.org/10.1038/s41467-022-35507-8)" *Nature Communications* doi: 10.1038/s41467-022-35507-8
* **Meng X**, **Liang Z**, **Dai X**, **Zhang Y**, Mahboub S, **Ngu DW**, Roston RL, **Schnable JC** (2021) "[Predicting transcriptional responses to cold stress across plant species.](https://doi.org/10.1073/pnas.2026330118)" *Proceedings of the National Academy of Sciences* doi: 10.1073/pnas.2026330118

### Funding Supporting Efforts in Comparative Genomics

* NSF [RoL: FELS: EAGER: Genetic Constraints on the Increase of Organismal Complexity Over Time.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1838307)
* USDA-NIFA [Identifying mechanisms conferring low temperature tolerance in maize, sorghum, and frost tolerant relatives.](https://portal.nifa.usda.gov/web/crisprojectpages/1008702-identifying-mechanisms-conferring-low-temperature-tolerance-in-maize-sorghum-and-frost-tolerant-relatives.html)

<a id="Phenotyping"></a>

## High Throughput Phenotyping


This area focuses on developing and deploying new algorithms, tools, and datasets for high throughput plant phenotyping. On the development side we collaborate closely with engineers and statisticians both here at UNL and at other universities around the world.

![CompGen](/images/optimized/Science_Images/HTPMerged.jpg)

### Recent Lab Publications on High Throughput Phenotyping

* **Shrestha N**, **Mangal H**, **Torres-Rodriguez JV**, **Tross MC**, **Lopez-Corona L**, **Linders K**, **Sun G**, **Mural RV**, **Schnable JC** (2025) "[Off-the-shelf image analysis models outperform human visual assessment in identifying genes controlling seed color variation in sorghum.](https://doi.org/10.1002/ppj2.70013)" *The Plant Phenome Journal* doi: 10.1002/ppj2.70013
* **Davis JM**, Gaillard M, **Tross MC**, **Shrestha N**, Ostermann I, **Grove RJ**, Li B, Benes B, **Schnable JC** (2025) "[3D reconstruction enables high-throughput phenotyping and quantitative genetic analysis of phyllotaxy.](https://doi.org/10.1016/j.plaphe.2025.100023)" *Plant Phenomics* doi: 10.1016/j.plaphe.2025.100023
* **Tross MC**, **Grzybowski M**, Jubery TZ, **Grove RJ**, **Nishimwe AV**, **Torres-Rodriguez JV**, **Sun G**, Ganapathysubramanian B, Ge Y, **Schnable JC** (2024) "[Data driven discovery and quantification of hyperspectral leaf reflectance phenotypes across a maize diversity panel.](https://doi.org/10.1002/ppj2.20106)" *The Plant Phenome Journal* doi: 10.1002/ppj2.20106

### Funding Supporting Efforts in High Throughput Phenotyping

* NSF [BTT EAGER: A wearable plant sensor for real-time monitoring of sap flow and stem diameter to accelerate breeding for water use efficiency.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1844563)
* ARPA-E: In-plant and in-soil microsensors enabled high-throughput phenotyping of root nitrogen uptake and nitrogen use efficiency.
* ARPA-E: Low cost wireless chemical sensor networks.
* USDA/NSF Joint Program [PAPM EAGER: Transitioning to the next generation plant phenotyping robots.](https://reeis.usda.gov/web/crisprojectpages/1011763-papm-eager-transitioning-to-the-next-generation-plant-phenotyping-robots.html)
* North Central Sun Grants: High throughput phenotyping to accelerate biomass sorghum improvement.
* Nebraska Corn Board: Genomes to Fields (G2F) - Predicting Final Yield Performance in Variable Environments.
* Wheat Innovation Foundation: A Low-Cost, High-Throughput Cold Stress Perception Assay for Sorghum Breeding.


<a id="QuantGen"></a>

## Quantitative Genetics


At its most basic, this area ensures that graduates of the Schnable lab are familiar with modern algorithms to perform basic quantitative genetic tasks including GWAS and genomic selection. However, more advanced work in this area involves developing and applying new statistical approaches that leverage the unique features of new phenotypic datasets (for example time-series data from whole mapping populations collected using HTP technologies) or sharing data across related species.

![CompGen](/images/optimized/Science_Images/QGMerged.jpg)

### Recent Lab Publications on Quantitative Genetics

* **Ali W**, **Grzybowski M**, **Torres-Rodriguez JV**, **Li F**, **Shrestha N**, **Mathivanan RK**, de Bernardeaux G, Hoang K, **Mural R**, Roston RL, **Schnable JC**, Sahay S (2025) "[Quantitative genetics of photosynthetic trait variation in maize.](https://doi.org/10.1093/jxb/eraf198)" *Journal of Experimental Botany* doi: 10.1093/jxb/eraf198
* **Torres-Rodriguez JV**, Li D, **Turkus J**, Newton L, **Davis J**, **Lopez-Corona L**, **Ali W**, **Sun G**, **Mural RV**, **Grzybowski M**, Zamft B, Thompson AM, **Schnable JC** (2024) "[Population level gene expression can repeatedly link genes to functions in maize.](https://doi.org/10.1101/2023.10.31.565032)" *The Plant Journal* doi: 10.1101/2023.10.31.565032
* **Sun G**, Yu H, Wang P, Lopez-Guerrero MG, **Mural RV**, **Mizero ON**, **Grzybowski M**, Song B, van Dijk K, Schachtman DP, Zhang C, **Schnable JC** (2023) "[A role for heritable transcriptomic variation in maize adaptation to temperate environments.](https://doi.org/10.1186/s13059-023-02891-3)" *Genome Biology* doi: 10.1186/s13059-023-02891-3

### Funding Supporting Efforts in Quantitative Genetics

* NSF [RII Track-2 FEC: Functional analysis of nitrogen responsive networks in Sorghum.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1826781&HistoricalAwards=false)
* FFAR [Crops in silico: Increasing crop production by connecting models from the microscale to the macroscale.](http://www.ncsa.illinois.edu/news/story/crops_in_silico_project_awarded_5_million)
* NSF [Center for Root and Rhizobiome Innovation.](https://nsf.gov/awardsearch/showAward?AWD_ID=1557417)
* ICRISAT: Application of tGBS And Genomic Selection to a Hybrid Pearl Millet Breeding Program.
